standards-catalogue/Rakefile

```diff
@@ -51,6 +60,24 @@ namespace :schema do
     end
     raise unless errors.all?
   end
+
+  desc "Validate organisations are schema compliant"
+  task :organisations do
```

Comment:
I'd say this can go under the `data_files` namespace, as it's related to data files

Reply:
done